apps/bitwarden_mailconfig.sh

```diff
@@ -207,8 +204,8 @@ then
 fi
 
 # Start Bitwarden
-start_if_stopped bitwarden
+check_command systemctl restart bitwarden
 msg_box "Your Bitwarden mailserver settings should be successfully changed by now.
-
+Bitwarden is now restarting. This can take a few minutes. Please wait until it is done.\n
```

Comment:
I think it would be smarter to add a check for the PID to go away, or to check the status of docker (stopped or running) and then continue.

Reply:
Maybe it is smarter but I currently don't know how to implement that...
Maybe you can have a look?

Reply:
https://stackoverflow.com/a/43723174

Reply:
Please go ahead 👍